core/modules/widgets/range.js

```diff
@@ -62,23 +64,88 @@ RangeWidget.prototype.render = function(parent,nextSibling) {
 
 RangeWidget.prototype.getValue = function() {
 	var tiddler = this.wiki.getTiddler(this.tiddlerTitle),
-		fieldName = this.tiddlerField || "text",
-		value   = this.defaultValue;
+		fieldName = this.tiddlerField,
+		value = this.defaultValue;
 	if(tiddler) {
 		if(this.tiddlerIndex) {
-			value = this.wiki.extractTiddlerDataItem(tiddler,this.tiddlerIndex,this.defaultValue || "");
+			value = this.wiki.extractTiddlerDataItem(tiddler,this.tiddlerIndex,this.defaultValue);
 		} else {
 			if($tw.utils.hop(tiddler.fields,fieldName)) {
 				value = tiddler.fields[fieldName] || "";
 			} else {
-				value = this.defaultValue || "";
+				value = this.defaultValue;
 			}
 		}
 	}
 	return value;
 };
 
+RangeWidget.prototype.prepareAttributes = function(options) {
```

Comment:
IMHO this name is not intuitive. In fact we are **processing** attributes to **prepare** variables, rather than preparing attributes.

Perhaps `getActionVariables()` so that the context is easier to undertand?

Reply:
`getActionVariables` would be more consistent with other usages in the core.